modules/workflows/OrcaCopyToGlacierWorkflow/orca_copy_to_glacier_workflow.asl.json

```diff
@@ -7,7 +7,7 @@
         "cma": {
           "event.$": "$",
           "task_config": {
-            "excludeFileTypes": "{$.meta.collection.meta.excludeFileTypes}",
+            "OrcaExcludedFileRegEx": "{$.meta.collection.orca.OrcaExcludedFileRegEx}",
             "s3MultipartChunksizeMb": "{$.meta.collection.meta.s3MultipartChunksizeMb}",
             "providerId": "{$.meta.provider.id}",
             "providerName": "{$.meta.provider.name}",

```

Comment:
There are likely other keys that can be moved to your new category. For example, "orcaDefaultBucketOverride" and "orcaDefaultStorageClassOverride".

Reply:
better would be to use the format for **o**rcaExcludedFileRegEx

Reply:
Taking a closer look at ORCA-290 and copy_to_glacier, it looks like the excluded file types aren't actually regexes... Am I missing something? Do we want to move the regex creation out of copy_to_glacier? @bhazuka ?

Reply:
They sort of are. See line https://github.com/nasa/cumulus-orca/blob/652d7a8ad32388cf94408956953de629e921f7db/tasks/copy_to_glacier/copy_to_glacier.py#L46 we are using the regex search capability and doing some of the regex construction on the fly. I am not sure what you mean by "moving the regex creation out of copy_to_glacier". If you are talking about the feature, the answer would be no as that is a needed feature used in both the ingest and recovery functionality. If you are talking about code placement the answer is probably but, that is likely a larger discussion as we should talk about other code layout changes and enhancements that we have tickets on the backlog for.

Reply:
will keep it as it is for now. Might want to move out the regex in the future if users want.

will rename to `orcaExcludedFileExtensions`
Also discuss with GHRC in working group if they want regex set up as well in excludeFileTypes key.